src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultClaimUidExtractor.cs

```diff
@@ -38,6 +38,17 @@ public string ExtractClaimUid(ClaimsIdentity claimsIdentity)
         // Internal for testing
         internal static IEnumerable<string> GetUniqueIdentifierParameters(ClaimsIdentity claimsIdentity)
```

Comment:
BTW `IsAuthenticated` is checked both just above (in `ExtractClaimUid()`) and in the `DefaultAntiforgeryTokenGenerator` calling code.